schduler/clock.py

```diff
@@ -0,0 +1,30 @@
+import os
+from rq import Queue
+from worker import conn
+import logging
+from apscheduler.schedulers.blocking import BlockingScheduler
+
+
+logger = logging.info(__name__)
+sched = BlockingScheduler()
+q = Queue(connection=conn)
+
+
+def trigger_updatedb():
+    os.system("python src/updatedb.py")
+
+
+def trigger_failures():
+    os.system("python tools/failures.py")
+
+
+@sched.scheduled_job('interval', hours=12)
+def timed_trigger_updatedb():
+    q.enqueue(trigger_updatedb())
+
+
+@sched.scheduled_job('interval', hours=6)
+def timed_trigger_failures():
+    q.enqueue(trigger_failures())
```

Comment:
there is a slight issue here, updatedb every 12 hours is fine, but if failures.py runs at the same time we will be off- it seems that failures.py only needs to run when updatedb is completed, otherwise it isn't getting new failures will be more out of date than needed.

Does hours == time_between_runs, or does hours == time of day?